### DISCUSSION TITLE
main content area is not expanding when you have a sidebar

### DISCUSSION THREAD

**Discussion by CoskunCakir** [diegomura/react-pdf#2230] [Q&A]
Hello,

I have a sidebar and main content area which looks like in the below REPL.

[REPL](https://tinyurl.com/2v3njyd7)

The problem is when the main content is more than 1 page, the area isn't expanding as much as the page width and it acts like there is a sidebar on the right, how can we make it expand along the page width?

Does anyone know why it's acting like this and have any idea how to fix it?

Thanks!

**Comment by CoskunCakir** [comment]
hey @jeetiss, I was wondering if you have an idea on this one

**Comment by jeetiss** [comment]
hello @coskuncakir it is expected behavior, i can't find the issue with the discussion, maybe later

checkout this layout https://react-pdf-repl.vercel.app/ it works as you want
